tests/collections/test_qnode_collection.py

```diff
@@ -249,7 +249,11 @@ def test_eval_tf(self, qnodes, skip_if_no_tf_support, parallel, interface):
         qc = qml.QNodeCollection([qnode1, qnode2])
         params = [0.5643, -0.45]
 
-        res = qc(params, parallel=parallel).numpy()
+        if parallel:
+            with pytest.warns(UserWarning, match="Parallel execution of QNodeCollections is an"):
+                res = qc(params, parallel=parallel).numpy()
```

Comment:
How come this was changed?

Reply:
There was a warning emitted for the parallel case. Since the warning is expected, I think it's a good idea to catch and check it.